wiki/Contests/SVS/1/en.md

```diff
@@ -0,0 +1,91 @@
+
+## Contest schedule
+
+| Event | Timestamp (UTC) |
+| --: | :-- |
+| Registration phase | 2021-07-01/2021-08-01 23:59 |
+| Submission phase | 2021-08-01/2021-09-01 23:59 |
+| Judging phase | 2021-09-01/2021-09-14 |
+| Results announcement | 2021-09-18 |
+
+## Organisation
+
+The Slider Velocity Showdown was run by various community members.
+
+| Position | Member(s) |
+| :-- | :-- |
+| Host | ![][flag_US] [Nifty](https://osu.ppy.sh/users/4956097), ![][flag_AR] [Axer](https://osu.ppy.sh/users/7299864), ![][flag_JP] [komasy](https://osu.ppy.sh/users/1980256) |
+| Designer | ![][flag_AR] [Atreevete](https://osu.ppy.sh/users/2615199) |
+| Judge | ![][flag_US] [Alchyr](https://osu.ppy.sh/users/4993032), ![][flag_GR] [Genjuro](https://osu.ppy.sh/users/3196091), ![][flag_JP] [iceOC](https://osu.ppy.sh/users/5482401), ![][flag_TW] [katacheh](https://osu.ppy.sh/users/6651672), ![][flag_DE] [Mew](https://osu.ppy.sh/users/2345156), ![][flag_BR] [Ppus](https://osu.ppy.sh/users/5918857), ![][flag_DE] [xMrtn-](https://osu.ppy.sh/users/866297) |
+| Translator | ![][flag_AR] [Axer](https://osu.ppy.sh/users/7299864) (Spanish translation), ![][flag_JP] [komasy](https://osu.ppy.sh/users/1980256) (Japanese translation) |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 4 months of osu!supporter |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 2 months of osu!supporter |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1361851)
+- [Discord server](https://discord.gg/8WbX3XdmTs)
+- [Results sheet](https://docs.google.com/spreadsheets/d/1ketuj4BgsevTpKVIM16d7drEdLYYYntPNdX2uIHfpk4/edit?usp=sharing)
+
+## Podium
+
+This competition has come to an end and resulted in the following podium:
+
+| Placing | User |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | ![][flag_NL] [Cookie_Tree](https://osu.ppy.sh/users/502722) |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | ![][flag_MY] [\[Zeth\]](https://osu.ppy.sh/users/9912966) |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | ![][flag_JP] [Grape_Tea](https://osu.ppy.sh/users/9540073) |
+
+## Ruleset
+
+### Contest rules
+
+- **The contest will consist of one round where contestants will map any song of their choosing.** The chosen song must not breach the osu! [song content rules](/wiki/Rules/Song_Content_Rules); if you wouldn’t upload it to your profile, then don’t use it here. Anything that violates the osu! rules will be disqualified.
+- **Each contestant is limited to a single difficulty of any level.**
```

Comment:
level?

Reply:
level as in difficulty level (kantan, futsuu, muzukashii, oni, inner oni), host probably worded it that way to avoid repeating the word "difficulty"

Reply:
then it'd be something like

```suggestion
- **Each contestant is limited to one of each difficulty level.**
```

Reply:
wouldn't that imply they have to map 5 diffs? one of each difficulty level? the contest takes only 1 diff, and it can be of any difficulty level.

Reply:
uh ok then i guess we'll just keep this wording